aircan/dags/api_ckan_load_gcp.py

```diff
@@ -2,16 +2,15 @@
 import time
 import json
 import ast
-from datetime import datetime
+from datetime import date, datetime
 
-# DAG Local import
-from aircan.dags.api_ckan_load_multiple_nodes import task_delete_datastore_table, task_create_datastore_table
```

Comment:
@hannelita So task_delete_datastore_table, task_create_datastore_table is moved from `dags` to this file? Any specific reason? I think we should keep those in the library.

Reply:
@shubham-mahajan I was reusing the same methods from the other DAG (multiple_nodes), but for now I'll rewrite these methods (they are essentially pretty different) and I'll probably delete the other DAG